src/components/stake/graph/StakingGraph.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+    <div class="chart-container">
+        <Vue3ChartJs
+            :id="chartData.id"
+            ref="chartRef"
+            :type="chartData.type"
+            :data="chartData.data"
+            :options="chartData.options" />
+    </div>
+</template>
+
+<script>
+import { defineComponent, ref } from '@vue/composition-api';
+import { Duration, DateTime } from 'luxon';
+import { formatSpaceyNumber } from '../../../lib/NumberFormatting';
+import { NIM_MAGNITUDE } from '../../../lib/Constants';
+import { i18n } from '../../../i18n/i18n-setup';
+import Vue3ChartJs from './lib/Vue3ChartJs.vue';
+
+const NOW = DateTime.now();
+
+const getProjectionLine = (apy, staked, steps, stepSize, dV = 1.13) => {
+    const projection = [];
+    const apm = (apy / 12.0) / 100.0;
+
+    for (let i = 0; i < steps; i++) {
+        projection.push(
+            {
+                x: NOW.plus(Duration.fromObject({ months: stepSize * i })).toMillis(),
+                y: staked * (1.0 + (apm * stepSize * i * (dV ** i))),
+            });
```

Comment:
I guess we'd usually format this as follows:
```suggestion
        projection.push({
            x: NOW.plus(Duration.fromObject({ months: stepSize * i })).toMillis(),
            y: staked * (1.0 + (apm * stepSize * i * (dV ** i))),
        });
```